modules/odc-using-the-devfile-registry.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: PROCEDURE
+[id="odc-using-the-devfile-registry_{context}"]
+= Using the Devfile Registry to access product Devfiles
+
+You can use the Devfiles in the  in the *+Add* flow of the *Developer* perspective to create an application. The Devfile registry provides a complete integration with the public community registry and the product registry. A devfile is a portable YAML file that describes your development environment without needing to configure it from scratch. Using the Devfile registry, you can use a pre-configured devfile to create an application.
```

Comment:
>The Devfile registry provides a complete integration with the public community registry and the product registry.

Should we have something like "The add flow provides a complete..." because the public registry and product registry are both devfile registries, so this sentence doesn't seem right to me.

Reply:
Also note that as of now, the devfile samples shown in dev catalog are fetched only from the community registry(https://registry.devfile.io), and not the RH registry(https://devfile-registry.redhat.com).